src/plugins/es_ui_shared/public/request/request.ts

```diff
@@ -21,13 +21,13 @@ import { useEffect, useState, useRef } from 'react';
 
 export interface SendRequestConfig {
   path: string;
-  method: string;
+  method: 'get' | 'post' | 'put' | 'delete' | 'patch';
```

Comment:
@sebelga I just noticed that we're missing the `HEAD` method here.

Reply:
Good point. Do you think we will ever need it?